src/functions.js

```diff
@@ -324,6 +324,38 @@ var _ = Mavo.Functions = {
 
 	len: text => str(text).length,
 
+	/**
+     * Search if a group, collection, or primitive contains needle
+	 * @returns Boolean if a haystack of object or primitive is passed
+	 * @returns Array of booleans if a haystack of array is passed
+     */
+    contains: (haystack, needle) => {
+		var type = $.type(haystack);
+		var ret = Mavo.Script.binaryOperation(haystack, needle, {
+			scalar: (haystack, needle) => {
+				if (type === "object") {
+					for (var property in haystack) {
+						ret = _.contains(haystack[property], needle);
+						if (Array.isArray(ret)) {
+							ret = Mavo.Functions.or(ret);
+						}
+						if (ret) { 
+							return true;
+						}
+					}
+				}
+				else if (type === "array") {
```

Comment:
Have you tested without this else if? `binaryOperation()` should handle this case...

Reply:
Yes! I deleted that block and nothing broke.